src/js/popup/popup.js

```diff
@@ -1775,6 +1776,36 @@
           link.addEventListener("click", popup.events.externalClick, false);
         });
       },
+      togglePwdMgrCompat: async () => {
+        const pwdMgrCompatPrefToggle = document.querySelector(".toggle-pwdmgr-compat");
+        
+        const stylePrefToggle = (userPref) => {
+          if (userPref === "pwdmgr-compat-enabled") {
+            pwdMgrCompatPrefToggle.classList.remove("pwdmgr-compat-disabled");
+            pwdMgrCompatPrefToggle.dataset.pwdMgrCompatPreference = "pwdmgr-compat-disabled";
+            return;
+          }
+          pwdMgrCompatPrefToggle.classList.add("pwdmgr-compat-disabled");
+          pwdMgrCompatPrefToggle.dataset.pwdMgrCompatPreference = "pwdmgr-compat-enabled";
+        };
+
+
+        const { pwdMgrCompat } = await browser.storage.local.get("pwdMgrCompat");
+
+        if (!pwdMgrCompat) {
+          browser.storage.local.set({ "pwdMgrCompat": "pwdmgr-compat-enabled" });
+          stylePrefToggle("pwdmgr-compat-enabled")
+        } else {
+          stylePrefToggle(pwdMgrCompat);
+        }
+
+        pwdMgrCompatPrefToggle.addEventListener("click", async(e) => {
+          const pwdMgrCompatPreference = e.target.dataset.pwdMgrCompatPreference;
+          await browser.storage.local.set({ "pwdMgrCompat" : pwdMgrCompatPreference });
+          stylePrefToggle(pwdMgrCompatPreference);
+        });
+        
+      },
```

Comment:
TODO: This function, along with `enableDataOptOut()` and `enableInputIconDisabling()` bascially all do the same. I would recommend re-factoring this to be DRY.